### DISCUSSION TITLE
Provide the path to download the ISO in the installation documentation

### DISCUSSION THREAD

**Discussion by brembelguebli** [scality/metalk8s#3234] [Ideas]
**Component**:

Metalk8s Documentation

**Why this is needed**:

The URL to download the ISO

**What should be done**:

Provide the URL for the ISO download, at least for the current version

**Implementation proposal** (strongly recommended):

**Test plan**:

**Comment by NicolasT** [comment]
We are not publicly distributing pre-built versions of MetalK8s and don't plan to do so. Hence, any such links must be in Scality-specific documentation, not in the open-source project documentation.
